kratos/tests/test_gid_io.py

```diff
@@ -3,10 +3,12 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import KratosMultiphysics
 from gid_output_process import GiDOutputProcess
+
+import testing_utilities as test_utils

```

Comment:
Is it doable to have testing_utilities as a sub module of KratosUnittest so we only import testing once?

Reply:
done